ecosystem_tests/ecosystem_tests_cli/commands/upload_plugin.py

```diff
@@ -30,11 +30,23 @@
 @prepare_test_env
 @ecosystem_tests.options.plugin_name
 @ecosystem_tests.options.plugin_version
-def upload_plugin(plugin_name, plugin_version):
+@ecosystem_tests.options.wagon_type
+def upload_plugin(plugin_name, plugin_version, wagon_type):
     """
     Upload wagon and yamls to Cfy Manager.
     """
     repo = 'cloudify-{}-plugin'.format(plugin_name)
+    legal_wagon_disto_names = [
+        'Centos Core', 'manylinux', 'Redhat Ootpa',
+        'Centos altarch', 'Redhat Maipo']
```

Comment:
This can be moved to a constant below imports section, e.g.:
```
import blah
from blahblah import blahblahblah

SOME_LIST = ['foo', 'bar', 'baz']

@prepare_test_env
def fn(....)
```